src/handlers/contacts_handler.py

```diff
@@ -26,25 +28,21 @@ def add_contact(args: list, book) -> str:
     return message
 
 
+@capitalize_name
 @input_error
-def change_contact(args: list, book) -> str:
+def change_contact(args: list, book: ContactBook) -> str:
     name, old, new = args
-    record = book.find(name)
+    contact = book.find(name)
 
-    if record:
-        for index in range(len(record.phones)):
-            if record.phones[index].phone == old:
-                try:
-                    record.phones[index] = Phone(new)
-                    return "Contact updated."
-                except PhoneVerificationError as e:
-                    return e.message
+    if contact:
+        contact.edit_phone(old, new)
 
-        return f"Contact {name} does not have entered phone number."
+        return f"Number changed successfully."
```

Comment:
please change to "Phone number successfully updated."

Reply:
Done